docs/docs/06-cli.md

```diff
@@ -16,8 +16,28 @@ $ npx memory-viz <path-to-file>
 
 replacing `<path-to-file>` with the path to a file containing MemoryViz-compatible JSON. If the file content is not compatible with MemoryViz, an error will be thrown.
 
-Optional arguments `--height` and `--width` can be used to specify the height and width of the generated SVG.
-
 ## Output
 
 The output is an SVG image generated by MemoryViz and the image is saved in the current working directory. The name of the SVG will be the same as that of the inputted file (i.e., if the inputted file is `david-is-cool.json`, the output will be `david-is-cool.svg`).
+
+## Options
+
+Below are optional arguments used to specify the way in which the SVG image is generated.
+
+#### `--height` and `--width`
```

Comment:
Use three `#`, not four. In general, subheadings should increase by one level (i.e., `#`) at a time

Reply:
also, make these two separate subheadings